distage/distage-model/src/main/scala/izumi/distage/model/definition/dsl/AbstractBindingDefDSL.scala

```diff
@@ -132,6 +193,33 @@ object AbstractBindingDefDSL {
     }
   }
 
+  final class MultipleRef(initial: SingletonBinding[DIKey.TypeKey], pos: SourceFilePosition, ops: mutable.Queue[MultipleInstruction] = mutable.Queue.empty) extends BindingRef {
+    override def interpret: collection.Seq[ImplBinding] = {
+      val (init, tags, refs) = ops.foldLeft((initial: SingletonBinding[DIKey.BasicKey], Set.empty[BindingTag], Seq.empty[SingletonBinding[DIKey]])) {
+        (acc, instr) =>
```

Comment:
@VladPodilnyk Could you use pattern matching here to match the unpack the accumulator instead of `_1`,`_2`,`_3` ?

Reply:
yup. give me a sec to fix this)

Reply:
Done.